src/Pidfile.jl

```diff
@@ -208,7 +208,7 @@ function open_exclusive(path::String;
         # now try again to create it
         file = tryopen_exclusive(path, mode)
         file === nothing || return file
-        wait(t) # sleep for a bit before trying again
+        fetch(t) # sleep for a bit before trying again
```

Comment:
Jameson didn't want this fixed: 
https://github.com/vtjnash/Pidfile.jl/pull/2#discussion_r209424875

Reply:
Oh, yikes, ok. I'll update this PR